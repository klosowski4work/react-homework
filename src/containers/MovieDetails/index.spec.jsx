import React from 'react';
import renderer from 'react-test-renderer';
import { MovieDetails } from './';

describe('MovieDetails component', () => {
    it('should render component', () => {
        const component = renderer.create(<MovieDetails />).toJSON()
        expect(component).toMatchSnapshot();
    });
});