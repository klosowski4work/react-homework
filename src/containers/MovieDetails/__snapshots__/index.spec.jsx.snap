// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetails component should render component 1`] = `"Movie details"`;
